worker-languagedetection/src/main/java/com/hpe/caf/worker/languagedetection/LanguageDetectionResultFormat.java

```diff
@@ -22,5 +22,13 @@
  */
 public enum LanguageDetectionResultFormat {
     SIMPLE,
-    COMPLEX;
+    COMPLEX,
+    COMPLEX_COMBINED,
+    COMPLEX_SPLIT;
+
+    public static boolean isComplexFormat(LanguageDetectionResultFormat formatToCheck)
```

Comment:
Use `final` when declaring parameters if the variable doesn't change in the function.